chainer/functions/noise/dropout.py

```diff
@@ -105,9 +106,28 @@ def dropout(x, ratio=.5, **kwargs):
             Input variable. A :math:`(s_1, s_2, ..., s_N)` -shaped float array.
         ratio (float):
             Dropout ratio. The ``ratio`` must be ``0.0 <= ratio < 1.0``.
+        mask (ndarray or None):
```

Comment:
This "ndarray" is not a link, but did we agree that that was okay?

Reply:
The discussion is ongoing in #4887. I added quote, so that we can later find it by ``grep -F 'ndarray`'``